manyfews/calculations/gefs.py

```diff
@@ -0,0 +1,201 @@
+from urllib.request import urlretrieve
+import pygrib
+import numpy as np
+from datetime import datetime, timedelta, timezone
+from .models import NoaaForecast
+from django.contrib.gis.geos import Point
+
+
+def GEFSdownloader(fileDate, forecastHour, latValue, lonValue):
+    """
+    This script is developed to download files, read files, and export necessary data for generating river flows.
+    Download from stp server:
+    here is an example link (01/02/2022):
+    "https://ftp.ncep.noaa.gov/data/nccf/com/gens/prod/gefs.20220201/00/atmos/pgrb2ap5/". There are many files in this
+    folder, and all are global at 0.25 degree resolution. The file naming system is:
+    gep30.t00z.pgrb2s.0p25.f240
+    gep = global ensemble prediction
+    30 = ensemble number. In this case it was the 30th member.
+    t00z = not sure, but they all have this code so we do not need to adjust for it.
+    pgrb2s = the number of weather forecast variables
+    0p25 = this means it is the 0.25 degree resolution model
+    f240 = number of hours into the future that the forecast is for.
+
+    :param fileDate: the select date of file with format: YYYYMMDD (only three days back)
+    :param fileName: the name of GEFS file. ( e.g. geavg.t00z.pgrb2a.0p50.f240)
```

Comment:
Should this be describing `forecastHour`? (I guess the parameter has changed since you added the doctoring.)

Reply:
yes, here is an introduction to the GEFS files' naming system. (maybe is not necessary to be added in the code). However, in order to make it more clear, I changed the comment to:    "f240 = number of hours into the future that the forecast is for, here is an example of the parameter 'forecastHour' = 240."

Reply:
But you are still adding documentation for parameter `fileName` which does not exist, and missing docs for parameter `forecastHour`